apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -316,8 +316,17 @@ NetSimLocalClientNode.prototype.sendMessage = function (payload, onComplete) {
 
   var localNodeID = this.myWire.localNodeID;
   var remoteNodeID = this.myWire.remoteNodeID;
+
+  // Who simulates?  Normally the receiving node
+  var simulatingNodeID = remoteNodeID;
+  // If sending to a router, we will do our own simulation
+  if (this.myRouter && this.myRouter.entityID === remoteNodeID) {
+    simulatingNodeID = localNodeID;
+  }
```

Comment:
Picking a simulating node #3:
When a client node sends a message it sets the recipient as the simulating node, unless the message is going to a router.